app/containers/App/tests/__snapshots__/index.test.js.snap

```diff
@@ -29,6 +29,6 @@ exports[`<App /> should render and match the snapshot 1`] = `
     />
   </Switch>
   <Footer />
-  <GlobalStyleComponent />
+  <UNDEFINED />
```

Comment:
This is weird ha? Something must be wrong.

Reply:
Looking at the [jest-styled-components issues](https://github.com/styled-components/jest-styled-components/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) there are some problems with the `v5`. Can you investigate more in this if you wanna merge your pull request. 

Side note:  Tests fail when you just add `import 'jest-styled-components'` into the `containers/App/tests/index.test.js` file which is discussed in the issues. Also there are issues on why classnames are changing too there. So I guess `--updateSnapshots` was a wrong suggestion after all. 

Can you look deeper into this?

Reply:
I noticed a change like this too, I think it was resolved with `jest-cli` v26